app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RestaurantsController < ApplicationController
+
```

Comment:
You don't need/want this extra linebreak here. (See https://github.com/bbatsov/ruby-style-guide#empty-lines-around-bodies)